tests/common.py

```diff
@@ -1,9 +1,9 @@
-from test_data.credentials import Credentials
+from tests.test_data.credentials import Credentials
 
 
 def login_steps(main_page):
     main_page.email_button.click()
     main_page.email_field.send_keys(Credentials.login_email)
     main_page.next_button.click()
     main_page.password_field.send_keys(Credentials.login_passw)
-    main_page.comein_button_click()
\ No newline at end of file
+    main_page.comein_button_click()
```

Comment:
> ...comein_button_click()
I'm not sure what did you mean ('common' or 'come in') but i suggest choosing one of these options:
 - ...come_in_button_click()
 - ...common_button_click()

Reply:
replaced pard of name "comein" -> "submit" (refactored)